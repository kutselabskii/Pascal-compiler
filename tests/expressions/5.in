1e15 and 2 - 'asd' + #123 / 0