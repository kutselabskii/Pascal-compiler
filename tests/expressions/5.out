|__ addition
    |-- substraction
    |   |-- bitwise_and
    |   |   |-- 1e+15
    |   |   |__ 2
    |   |__ asd
    |__ division
        |-- {
        |__ 0
